infrastructure/environments/preview/variables.tf

```
@@ -44,6 +44,12 @@ variable "db_name" {
   default     = "qfdmo"
 }
 
+variable "db_wh_name" {
```

```suggestion
variable "wh_db_name" {
```

je chipote un peu, mais je préfère la notion de préfixe, on pourrait tout namespacer `wh_` pour ce qui relève du warehouse